includes/text-snippets/ca_supported_chains.md

```diff
@@ -1,14 +1,17 @@
-<!---
+
 * Arbitrum
 * Base
 * Ethereum
 * HyperEVM
 * Kaia
-* Linea
+* ~~Linea~~
```

Comment:
@shaloo Let's just leave Linea out instead of explicitly striking it out please. Nobody is using it anyway (we were having issues with their network).

Reply:
ok

Reply:
It will still be mentioned in older release notes where we added Linea support for the first time. So just removing it in this list without crossing can still need explanations. JFYI